README.md

```diff
@@ -100,6 +101,18 @@ $ less /var/log/puppetlabs/puppetdb/puppetdb-access.log
 127.0.0.1 - - [01/Nov/2022:15:56:21 +0000] "POST /pdb/query/v4 HTTP/1.1" 200 1793 "-" "HTTPClient/1.0 (2.8.3, ruby 2.7.6 (2022-04-12))" 99 21 -
 ```
 
+## Logs
+
+By default, this module will create the log file under
+
+    /var/log/puppetlabs/bolt-server/bolt-server.log
+
+The log level is set to 'debug'. For more information, please read the Bolt logs doc [here](https://puppet.com/docs/bolt/latest/logs.html#log-levels)
```

Comment:
Do we really want to make `debug` the default log level?

Reply:
Maybe not!  @timidri  Should I change that to `"info"`?

Reply:
Maybe that is the safe choice, yes, @mcka1n .

Reply:
@timidri Makes sense to me, I pushed the change based on this feedback!

Thanks!

Reply:
Great stuff, only it also needs to change on line 110 :-)